model_security_adjust_cft/views/stock_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+ <record model="ir.ui.view" id="view_stock_quant_form_z161212">
+        <field name="name">stock.quant.form.z1212</field>
+        <field name="model">stock.quant</field>
+        <field name="inherit_id" ref="stock.view_stock_quant_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='inventory_value']" position="replace">
+                <field name="inventory_value" groups="account.group_account_manager"/>
```

Comment:
You missed this one.

Why don't you make the style consistent with others? i.e. use `position="attributes"` instead of `replace`.